app/assets/stylesheets/smart_answers.scss

```diff
@@ -40,5 +40,13 @@
         padding: govuk-spacing(2);
       }
     }
+
+    // stylelint-disable selector-max-id
+    & > #content > .govuk-grid-row > :only-child {
```

Comment:
We could skip the #content id and first child selector, and instead do this more generalised descendent selector:

`& .govuk-grid-row > :only-child`

...which would work with our current HTML where a single column is inside a single row:

```
<div class="smart-answer">
  <main id="content" role="main">
    <div class="govuk-grid-row">
      <div class="govuk-grid-column-two-thirds">
          CONTENT
      </div>
    </div>
  </main>
</div>
```

But if there's the possibility of multiple rows being used in this markup, the descendent selector might also get matched when we don't want it to. 

So I'm in favour of keeping the extra ID + child selector combination for more accurate matching, but perhaps it's better to change it to use the `main` selector rather than an ID, thus:

`& > main > .govuk-grid-row > :only-child {`

What do you think?

Reply:
That's fair, thanks for the explanation. Fine to leave as-is for now.